src/envoy/http/service_control/client_cache.h

```diff
@@ -113,6 +117,8 @@ class ClientCache : public Envoy::Logger::Loggable<Envoy::Logger::Id::filter> {
 
   // Used to retrieve the current time for tracing.
   Envoy::TimeSource& time_source_;
+
+  friend class test::ClientCacheCheckResponseTest;
```

Comment:
Per style guide,  friend class should be right after "private"

Reply:
Done. I did not see this in the style guide. But it seems to be a common recommendation.

https://google.github.io/styleguide/cppguide.html#Friends